packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.ts

```diff
@@ -304,6 +304,7 @@ export class SavedObjectsRepository implements ISavedObjectsRepository {
       migrationVersion,
       coreMigrationVersion,
       typeMigrationVersion,
+      managed,
```

Comment:
No SOR changes were required for `bulkCreate`?

Reply:
Great catch! I was assuming migrations would take care of setting `managed` and changed it while working on adding the option to `import`. What we _actually_ need (and want) is to set the default for any objects created as of now. I'll double check `create` too.